tracer/src/Datadog.Trace/Iast/TruncationUtils.cs

```diff
@@ -33,4 +33,22 @@ public static void WriteTruncatableValue(this JsonWriter writer, string? value,
             writer.WriteValue(value);
         }
     }
+
+    public static void InsertTruncableValue(Dictionary<string, object> dict, string key, string? value, int maxValueLength)
```

Comment:
If `maxValueLength` is `0`, this method will not truncate `value`. Was that intentional?

Reply:
This PR don't change this implementation. I guess when the value is 0, it means that the truncation is disabled.

Reply:
Got it, thanks!